tokenizers/src/decoders/mod.rs

```diff
@@ -98,7 +208,7 @@ mod tests {
         match parse {
             Err(err) => assert_eq!(
                 format!("{err}"),
-                "data did not match any variant of untagged enum DecoderWrapper"
+                "data did not match any variant of untagged enum DecoderUntagged"
```

Comment:
We could change the custom to properly output decoder wrapper

Reply:
I think the new message is better, since `Untagged` is more descriptive than wrapper (both mean nothing to regular users, but at least untagged points to the fact that the type is missing.

We can override the message as needed. Do you have a good suggestion for the message?

Reply:
Out of the box, something like "The `type` field is not present and we could not load into any of the variants of DecoderUntagged"